src/controllers/movieController.ts

```diff
@@ -0,0 +1,160 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+import config from '../config';
+import logger from '../utils/logger';
+import HttpService from '../services/httpService';
+import StorageService, { StorageError } from '../services/storageService';
+import { CreateRequestBody, GetAllRequestQuery, SortOrder } from './types';
+import { FakeMovieListType } from '../data/fakeMovieList';
+import { customComparer } from './helpers';
+
+class MovieController {
+  get = async (req: Request, res: Response) => {
```

Comment:
please check out Typescript class' methods syntax
https://www.typescriptlang.org/docs/handbook/classes.html

Reply:
If you are pointing here to arrow function syntax this is a valid one according to documentation https://www.typescriptlang.org/docs/handbook/2/classes.html#arrow-functions. I used such approach because of loosing 'this' in calling context. After refactoring of applyValidationResult method hopely I will get back to classic method declaration.